### ISSUE TITLE
Training only working without pre-trained data

### ISSUE THREAD

**Issue by ChipsSpectre** [zylo117/Yet-Another-EfficientDet-Pytorch#241]
Hello
first, thank you for the great repository you created!
I tested the *efficientdet_test_videos.py* on my custom dataset and get decent detections from the pretrained models.
When I train from scratch, the network is also learning useful bounding boxes.

However, when I use the pretrained weights, the network predicts a multitude of bounding boxes with confidence of *1.0* / even in those areas that are clearly not related to the objects.

I use the same normalization as for Coco and apply only cutting and isotropic scaling for data augmentation.

Do you have any idea what causes this problem? / Have others experienced this phenomenon, too?
